src/mapnode.h

```diff
@@ -250,7 +250,7 @@ struct MapNode
 	/*
 		Gets list of collision boxes
 	*/
-	std::vector<aabb3f> getCollisionBoxes(INodeDefManager *nodemgr) const;
+	std::vector<aabb3f> getCollisionBoxes(INodeDefManager *nodemgr, u8 neighbors = 0) const;
```

Comment:
Done, and I did ```getSelectionBoxes()``` as well, as well as a few calls to those outside the connected fence code path.

However, these 3 functions are now no longer ```const```. Is that OK? Or do we want nodemgr to be ```const``` again?